src/main/java/org/example/C03_10_PatternRightAngleTriangle.java

```diff
@@ -0,0 +1,19 @@
+package org.example;
+
+import java.util.Scanner;
+
+public class C03_10_PatternRightAngleTriangle {
```

Comment:
Rename to "ConsecutiveNumbersPatterRightAngleTriangle"

Reply:
done